src/studio/src/designer/backend/Controllers/DatamodelsController.cs

```diff
@@ -56,6 +56,14 @@ public async Task<IActionResult> UpdateDatamodel(string org, string app, string
                 JsonValue jsonValue = await JsonValue.ParseAsync(textReader);
                 JsonSchema jsonSchemas = new Manatee.Json.Serialization.JsonSerializer().Deserialize<JsonSchema>(jsonValue);
 
+                // Create the directory if it does not exist
+                var appPath = _repository.GetAppPath(org, app);
```

Comment:
We try to avoid using var is the type is know. 
Maybe set to string for both this and the line below?